conftest.py

```diff
@@ -23,6 +23,8 @@
 
 
```

Comment:
It's interesting that Amazon seems to use pytest to drive running tests of the aws-fpga stuff.